addons/missionTesting/functions/fnc_displayMenu.sqf

```diff
@@ -0,0 +1,260 @@
+                _ctrlCreateText ctrlSetStructuredText parseText _text;
+                _ctrlCreateText ctrlSetBackgroundColor [1, 1, 1, 0.25];
+                _ctrlCreateText ctrlSetPosition [0.01,GVAR(yStartCoord),0.77,CONTROL_SIZE_H];
+                _ctrlCreateText ctrlCommit 0;
+                _ctrlHeight = ctrlTextHeight _ctrlCreateText;
+                _ctrlCreateText ctrlSetPosition [0.01,GVAR(yStartCoord),0.77,_ctrlHeight + 0.01];
+                _ctrlCreateText ctrlCommit 0;
+                //Create check box and add eventhandler that updates the Master Array when changed.
+                _ctrlCreateCB = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscCheckBox),_idc,CONTROL_GROUP_L];
+                _ctrlCreateCB ctrlSetPosition [0.78,GVAR(yStartCoord),0.05,CONTROL_SIZE_H];
+                _ctrlCreateCB ctrlCommit 0;
+                _ctrlCreateCB cbSetChecked _checked;
+                _ctrlCreateCB ctrlAddEventHandler [QUOTE(CheckedChanged),{_this call FUNC(updateCheckedCB);}];
+                //Set new value of yStartCoord so that the next line is created at the right point below the previous line.
+                GVAR(yStartCoord) = GVAR(yStartCoord) + _ctrlHeight + 0.02;
+            };
+        } forEach _sectionArray;
+
+        private _createCtrlLine2 = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscLine),-1,CONTROL_GROUP_L];
+        _createCtrlLine2 ctrlSetPosition [0.01,GVAR(yStartCoord),LINE_W,0];
+        _createCtrlLine2 ctrlCommit 0;
+        INCREMENT_YCOORD;
+
+        private _ctrlCreateSectionPassQuestion = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscText),-1,CONTROL_GROUP_L];
+        private _sectionQuestionPassFail = format ["Does this mission pass the %1 requirements?",_sectionTitle];
+        _ctrlCreateSectionPassQuestion ctrlSetText _sectionQuestionPassFail;
+        _ctrlCreateSectionPassQuestion ctrlSetPosition [0.01,GVAR(yStartCoord),0.7,CONTROL_SIZE_H];
+        _ctrlCreateSectionPassQuestion ctrlCommit 0;
+
+        private _idcPassFail = IDC_CBITEMS + _forEachIndex;
+        private _ctrlCreateSectionInstruction = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscCombo),_idcPassFail,CONTROL_GROUP_L];
+        lbCLear _ctrlCreateSectionInstruction;
+        {
+            _ctrlCreateSectionInstruction lbAdd _x;
+        } forEach A_PASSFAIL;
+        _ctrlCreateSectionInstruction lbSetCurSel _sectionPassFail;
+        _ctrlCreateSectionInstruction ctrlSetPosition [0.70,GVAR(yStartCoord),0.12,CONTROL_SIZE_H];
+        _ctrlCreateSectionInstruction ctrlAddEventHandler [QUOTE(LBSelChanged),{_this call FUNC(updateSelectionLB);}];
+        _ctrlCreateSectionInstruction ctrlCommit 0;
+
+        INCREMENT_YCOORD_TEXT;
+        private _createCtrlLine3 = DISPLAY_TESTMENU ctrlCreate [QUOTE(RscLine),-1,CONTROL_GROUP_L];
+        _createCtrlLine3 ctrlSetPosition [0.01,GVAR(yStartCoord),LINE_W,0];
+        _createCtrlLine3 ctrlCommit 0;
+        INCREMENT_YCOORD;
+
+        if(_missionMaker != name player) then {
```

Comment:
use `ACE_PLAYER` so if they try to complete this while remote controlled their name won't be of the AI's

Reply:
Replaced this through out. Wasn't aware of this. Thank you.